error not being wrapped correctly

I am seeing a panic from an error that does not look like it has been wrapped correctly
```
$ sudo -E localizer --namespace test
INFO[0000] using apiserver REDACTED
INFO[0000] using cluster domain: cluster.local          
INFO[0000] using ip cidr: 127.0.0.1/8                   
INFO[0000] checking if an instance of localizer is already running 
WARN[0010] failed to contact existing instance, cleaning up socket 
INFO[0010] starting GRPC server on unix:///var/run/localizer.sock 
INFO[0010] Waiting for caches to sync...                
INFO[0011] Caches synced                                
INFO[0011] Starting 1 proxier worker(s)                  component=proxier service="*api.GRPCServiceHandler"
panic: errors: *target must be interface or implement error

goroutine 169 [running]:
errors.As({0x1025aeb00, 0x140001fe140}, {0x1024ba960, 0x140001fe180})
        /home/circleci/.asdf/installs/golang/1.19.11/go/src/errors/wrap.go:89 +0x3cc
github.com/pkg/errors.As(...)
        /home/circleci/.asdf/installs/golang/1.19.11/packages/pkg/mod/github.com/pkg/errors@v0.9.1/go113.go:31
github.com/getoutreach/localizer/internal/proxier.(*worker).stopPortForward(0x14000ca0000, {0x1025caa08, 0x140006bf340}, 0x1400047a000)
        /home/circleci/project/internal/proxier/portforward.go:427 +0x14c
github.com/getoutreach/localizer/internal/proxier.(*worker).CreatePortForward.func1()
        /home/circleci/project/internal/proxier/portforward.go:278 +0x48
github.com/getoutreach/localizer/internal/proxier.(*worker).CreatePortForward(0x14000ca0000, {0x1025caa08?, 0x140006bf340}, 0x140003162a0)
        /home/circleci/project/internal/proxier/portforward.go:297 +0x1064
github.com/getoutreach/localizer/internal/proxier.(*worker).Start(0x14000ca0000, {0x1025caa08, 0x140006bf340})
        /home/circleci/project/internal/proxier/portforward.go:168 +0x100
created by github.com/getoutreach/localizer/internal/proxier.NewPortForwarder
        /home/circleci/project/internal/proxier/portforward.go:131 +0x350
```

--- a/internal/proxier/portforward.go
+++ b/internal/proxier/portforward.go
@@ -423,7 +423,7 @@ func (w *worker) stopPortForward(ctx context.Context, conn *PortForwardConnectio
 			args := []string{"lo0", "-alias", ipStr}
 			if err := exec.Command("ifconfig", args...).Run(); err != nil {
 				message := ""
-				var exitError exec.ExitError
+				var exitError *exec.ExitError
 				if ok := errors.As(err, &exitError); ok {
 					message = string(exitError.Stderr)
 				}

